SwiftUIStarter/SwiftUIStarter/NativeDisplayView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  NativeDisplayUIView.swift
+//  SwiftUIStarter
+//
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+import SwiftUI
+import CleverTapSDK
+
+struct NativeDisplayUIView: View {
```

Comment:
Rename to NativeDisplayView as the name of the file and other View names.
It might be better to prefix with `CT` as well.